toolkit/scripts/toolchain.mk

```diff
@@ -296,26 +292,24 @@ $(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(STATUS_FLAGS_DIR)/toolchain_local_tem
 
 # No archive was selected, so download from online package server instead. All packages must be available for this step to succeed.
 else
-$(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(STATUS_FLAGS_DIR)/daily_build_auto_cleanup.flag $(depend_REBUILD_TOOLCHAIN) $(go-downloader)
-	@rpm_filename="$(notdir $@)" && \
-	rpm_dir="$(dir $@)" && \
-	log_file="$(toolchain_downloads_logs_dir)/$$rpm_filename.log" && \
-	echo "Downloading toolchain RPM: $$rpm_filename" | tee -a "$$log_file" && \
-	mkdir -p $$rpm_dir && \
-	cd $$rpm_dir && \
-	for url in $(PACKAGE_URL_LIST); do \
-		$(go-downloader) --no-verbose --no-clobber $$url/$$rpm_filename \
-			$(if $(TLS_CERT),--certificate=$(TLS_CERT)) \
-			$(if $(TLS_KEY),--private-key=$(TLS_KEY)) \
-			--log-file $$log_file 2>/dev/null && \
-		echo "Downloaded toolchain RPM: $$rpm_filename" >> $$log_file && \
-		touch $@ && \
-		break; \
-	done || { \
-		echo "\nERROR: Failed to download toolchain package: $$rpm_filename." && \
-		echo "ERROR: Last $(toolchain_log_tail_length) lines from log '$$log_file':\n" && \
-		tail -n$(toolchain_log_tail_length) $$log_file | sed 's/^/\t/' && \
-		$(call print_error,\nToolchain download failed. See above errors for more details.) \
+$(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(STATUS_FLAGS_DIR)/daily_build_auto_cleanup.flag $(depend_REBUILD_TOOLCHAIN) $(go-downloader) $(SCRIPTS_DIR)/toolchain/download_toolchain_rpm.sh
+	@log_file="$(toolchain_downloads_logs_dir)/$(notdir $@).log" && \
+	rm -f "$$log_file" && \
+	$(SCRIPTS_DIR)/toolchain/download_toolchain_rpm.sh \
+		--downloader-tool "$(go-downloader)" \
+		--rpm-name "$(notdir $@)" \
+		--dst "$@" \
+		--log-base "$$log_file" \
```

Comment:
is this intentional? (`$$log_file`)

Reply:
Yes, its referencing the bash variable `$log_file` but needs an escape for Make.